packages/orbit-components/src/Wizard/Wizard.ct.tsx

```diff
@@ -1,76 +1,79 @@
 import * as React from "react";
 import { test, expect } from "@playwright/experimental-ct-react";
+import { defaultTokens } from "@kiwicom/orbit-design-tokens";
 
 import RenderInRtl from "../utils/rtl/RenderInRtl";
 import { WizardStory, WizardStoryHover } from "./Wizard.ct-story";
 
 test.describe("visual Wizard", () => {
-  async function maybeHoverByText(component, text: string) {
-    if ((await component.getByText(text).count()) === 0) {
-      return Promise.resolve();
-    }
-
-    return component.getByText(text).hover();
-  }
+  const { breakpointLargeMobile } = defaultTokens;
```

Comment:
I had to polish this visual test due to unexpected failing for small/medium mobile views in `row/column hover completed step` block. I know it's out of scope of React 19 upgrade ticket, however, it helped me to solve that issue.